src/index.js

```diff
@@ -175,7 +179,7 @@ class DropdownTreeSelect extends Component {
   render() {
     const dropdownTriggerClassname = cx({
       'dropdown-trigger': true,
-      arrow: true,
+      arrow: !this.props.disabled,
```

Comment:
I think we should leave the arrow as is but add another class `disabled` for `disabled` case. This accomplishes few things:

 - keeps the current behavior as is
 - the arrow should not disappear, rather appear as disabled for a better UX
 - allows users to extend or style differently when the control is disabled (since it's not a native control, they cannot use CSS attribute selector)

Also, I think we need some disabled styling for arrow like greying it out and setting cursor to not-allowed.